Makefile

```diff
@@ -69,24 +69,24 @@ clean:
 	rm -rf ./testbin; rm -rf ./bin
 
 build: generate fmt vet cluster-config ## Build manager binary.
-	go build -o bin/arlon main.go
+	go build --ldflags="-s -w" -o bin/arlon main.go
```

Comment:
- why not just use $(LD_FLAGS) here
- why not add -s and -w to LD_FLAGS?
- what do -s and -w do? Your PR description needs some content.

Reply:
Added it to the PR description and made the requested changes